bear-requirements.yaml

```diff
@@ -172,10 +172,6 @@ pip_requirements:
     version: ~=3.4.0
   lxml:
     version: ==3.6.0
-  memento_client:
```

Comment:
But it looks like those libraries are being used. Could you post the command how you get these removals?

Even if the removal is necessary, IMO you should create an issue instead of solving it in this pr. At least you should split it into another commit, since this is unrelated to your current commit.

Reply:
This is not addressed.